src/Form/EventListener/AnyNoneFilterListener.php

```diff
@@ -45,26 +48,44 @@ class AnyNoneFilterListener
      */
     protected $columnsExtractor;
 
+    /**
+     * Elements (listing entities) containing AnyNoneSelection
+     *
+     * @var string[]
+     */
+    private $entityElementsWithAnyNone = array();
+
     public function __construct(ColumnsExtractor $columnsExtractor)
     {
         $this->columnsExtractor = $columnsExtractor;
     }
 
+    /**
+     * Adds data for AnyNoneFilter to the form (events, ...).
+     *
+     * @param \CubeTools\CubeCommonBundle\Form\EventListener\FormBuilderInterface $builder
+     */
+    public function addToBuilder(FormBuilderInterface $builder)
+    {
+        $builder->addEventListener(FormEvents::PRE_SET_DATA, array($this, 'addAnyNoneColumns'));
+        $builder->addEventListener(FormEvents::PRE_SUBMIT, array($this, 'processAnyNoneColumns'));
+    }
+
     /**
      * Method to be called as PRE_SET_DATA listener for form.
      *
+     * Add fields to the form to submit data to frontend.
+     *
      * @param \Symfony\Component\Form\FormEvent $event
      */
     public function addAnyNoneColumns(FormEvent $event)
     {
-        $event->getForm()->add(self::KEY_ANY_NONE_COLUMNS, HiddenType::class, array(
-            'data' => json_encode($this->columnsExtractor->getEntitiesColumnsByName($event->getForm())),
-        ));
-        $event->getForm()->add(self::KEY_ANY_NONE_SELECTED_COLUMNS, HiddenType::class);
+        $this->addAnyNoneFields($event->getForm());
```

Comment:
It's the only execution of that method, why the content of this method is not here? What's more, method addAnyNoneFields is private.

Reply:
because of #115. Agree that it does not make sense in this pr.

Reply:
sorry, is used in #115 in addToBuilder()

But this use will probably go away...

Reply:
private method is removed now